coins.json

```diff
@@ -960,14 +960,14 @@
     "symbol": "BLZ",
     "decimals": 6,
     "blockchain": "Cosmos",
-    "derivationPath": "m/44'/118'/0'/0/0",
+    "derivationPath": "m/44'/483'/0'/0/0",
```

Comment:
<img width="870" alt="Screen Shot 2021-06-19 at 8 10 08 AM" src="https://user-images.githubusercontent.com/1641795/122646763-c49a7380-d0d5-11eb-9fd8-8fec3f1b0d6b.png">

https://coinmarketcap.com/currencies/bluzelle/

BNT vs BLZ, what's the correct one?

Reply:
BNT is a legacy reference and currently used only in internal reference. BLZ will be symbol for enduser prespective.

Reply:
We will update [slip's reference](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) soon.